src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    private String details;
+
+    public Event(String taskName, String details) {
+        super(taskName);
+        this.details = details;
+    }
+
+    @Override
+    public String toString() {
+        String eventTag = "[E]";
```

Comment:
Since the eventTag is associated with all tasks, perhaps we can put that as a private attribute associated with the subclasses of Tasks? I would suggest you introduce it as a class attribute of Task and assign it with the corresponding tag in the constructors of the subclasses.